config/index.js

```diff
@@ -285,5 +288,46 @@ module.exports = {
     'wrap-regex': 'error',
     'yield-star-spacing': ['error', 'both'],
     'yoda': ['error', 'never'],
+
+    'import/default': 'error',
+    'import/dynamic-import-chunkname': 'off',
+    'import/export': 'error',
+    'import/exports-last': 'off',
+    'import/extensions': ['error', 'never', { 'json': 'always' }],
+    'import/first': 'error',
+    'import/group-exports': 'off',
+    'import/max-dependencies': 'off',
+    'import/named': 'error',
+    'import/namespace': 'error',
+    'import/newline-after-import': ['error', { 'count': 1 }],
+    'import/no-absolute-path': 'error',
+    'import/no-amd': 'off',
+    'import/no-anonymous-default-export': ['error', { 'allowObject': true }],
+    'import/no-commonjs': 'off',
+    'import/no-cycle': 'off',
```

Comment:
Do we have any cycles? This seems like a rule we should use.

Reply:
It's "computationally expensive" and rarely cycles are fine. Those two things lead me to think that this isn't worth turning on.

Reply:
Hmm, OK. We can always turn it on later anyway.